CIS560-RecipeManager/CIS560-RecipeManager/Pantry/partialQuery_Ingredient.cs

```diff
@@ -0,0 +1,118 @@
+﻿/**
+ * partialQuery_Ingredient.cs
+ * partial Query class; specific to Adding Ingredients to the database
+ * 
+ * author: Project Team 1 - CIS 560 KSU Fall 2018
+ * edited by: David K. Hwang
+ * 
+ */
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
From my understanding, all of the partial classes need to be in the same namespace, so we should put all of the IQuery partial classes in the Repository folder/namespace.